Valet.xcodeproj/project.pbxproj

```diff
@@ -497,6 +639,106 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		26E682811BA8B3F900EFF4EA /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
We can strip these settings out too, and fall back on Xcode's brains. These are just baseline targets that I generated by creating new Cocoa and Cocoa Touch Framework targets in Xcode. Let me know if you think we should just rip them out!